README.md

```diff
@@ -1,31 +1,95 @@
-# perseo
+# Perseo Context-Aware CEP
 
-[![FIWARE Processing](https://nexus.lab.fiware.org/static/badges/chapters/processing.svg)](https://www.fiware.org/developers/catalogue/)
-[![License: APGL](https://img.shields.io/github/license/telefonicaid/perseo-fe.svg)](https://opensource.org/licenses/AGPL-3.0)
-<br/>
-[![Build badge](https://img.shields.io/travis/telefonicaid/perseo-fe.svg)](https://travis-ci.org/telefonicaid/perseo-fe/)
-[![Coverage Status](https://coveralls.io/repos/github/telefonicaid/perseo-fe/badge.svg?branch=master)](https://coveralls.io/github/telefonicaid/perseo-fe?branch=master)
+[![FIWARE Processing, analysis and visualization of context information](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/visualization.svg)](https://www.fiware.org/developers/catalogue/)
+[![License: AGPLv3+](https://img.shields.io/badge/License-AGPLv3+-blue.svg)](./LICENSE)
+[![Docker Status](https://img.shields.io/docker/pulls/fiware/perseo.svg)](https://hub.docker.com/r/fiware/perseo/)
+[![Support badge](https://img.shields.io/badge/tag-fiware--perseo-orange.svg?logo=stackoverflow)](https://stackoverflow.com/questions/tagged/fiware-perseo)
+<br> [![Documentation badge](https://img.shields.io/readthedocs/perseo.svg)](https://perseo.readthedocs.io/en/latest/)
+[![Build Status](https://travis-ci.org/telefonicaid/perseo-fe.svg?branch=master)](https://travis-ci.org/telefonicaid/perseo-fe)
 ![Status](https://nexus.lab.fiware.org/static/badges/statuses/perseo.svg)
+[![Swagger Validator](https://img.shields.io/swagger/valid/2.0/https/raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/json/petstore-expanded.json.svg)](https://app.swaggerhub.com/apis/smartsdk/ngsi-tsdb)
 
-CEP implementation for IoT platform
+## Overview
 
-## Index
+Perseo is a Complex Event Processing (CEP) software designed to be fully _NGSIv2_-compliant. It uses NGSIv2 as the
+communication protocol for events, and thus, Perseo is able to seamless and jointly work with _context brokers_ such as
```

Comment:
Is there any other Context Broker on Earth apart from Orion :) ? I'd suggest an sligth change in the wording. From:

> Perseo is able to seamless and jointly work with _context brokers_ such as [Orion](https://github.com/telefonicaid/fiware-orion)

to

> Perseo is able to seamless and jointly work with [Orion Context Broker](https://github.com/telefonicaid/fiware-orion)

Reply:
Jejeje. ... yes, there are other Context brokers in the world and other services providing Context Broker compatible APIS. :)

Reply:
With which other CB/services apart from Orion has been Perseo tested?

Reply:
> With which other CB/services apart from Orion has Perseo been tested?

This comment is still opened. The answer to that question wasn't provided...